lua/astrocommunity/pack/python/init.lua

```diff
@@ -75,6 +75,9 @@ return {
   },
   {
     "linux-cultist/venv-selector.nvim",
+    branch = "main",
```

Comment:
This shouldnt be nessecary at all. I am using venv-selector without defining the branch.